syft/pkg/url.go

```diff
@@ -10,6 +10,7 @@ import (
 
 const (
 	PURLQualifierArch   = "arch"
+	PURLQualifierCPES   = "cpes"
```

Comment:
could use a test around CPEs as a qualifier (as horrifying as using it sounds 😳 )